blog.md

```diff
@@ -0,0 +1,106 @@
+```
+
+Now we are ready to look at the user inputs, file structure, and run the script.
+
+Input parameters and directory setup
+------------------------------------
+
+Open the main script, `batch.R`. There are some basic instructions at the top. Below that are the user inputs, set up for the included example data. The user supplies information about input/output folder names and locations, constituents, load units, and load rate units. The consituent names need to the match the names of the input subfolders that contain the input data (paired water quality and discharge measurements). The discharge folder, containing the discharge measurements used to make the load predictions, works the same way. `siteInfo` is a .csv file (inside `inputFolder`) that contains metadata for water quality and discharge sites. Look at the example file (`three_ANA_sites/siteInfo.csv`) for reference:
+
+    ##   matching.site   site.id site.name lat lon basin.area constituent   units
+    ## 1     RONC02800 RONC02800      Ronc   1   1        100         NO3 mg L^-1
+    ## 2     RONC02800 RONC02800      Ronc   1   1        100          PT mg L^-1
+    ## 3     RONC02800 RONC02800      Ronc   1   1        100           Q     cms
+    ## 4     MOGU02900 MOGU02900      Mogu   2   2        200         NO3 mg L^-1
+    ## 5     MOGU02900 MOGU02900      Mogu   2   2        200          PT mg L^-1
+    ## 6     MOGU02900 MOGU02900      Mogu   2   2        250           Q     cms
+    ## 7     ORIZ02900 ORIZ02900      Oriz   3   3        300         NO3 mg L^-1
+    ## 8     ORIZ02900 ORIZ02900      Oriz   3   3        300          PT mg L^-1
+    ## 9     ORIZ02900 ORIZ02900      Oriz   3   3        300           Q     cms
+
+Here is input consituent data formatting:
+
+    ##         date         Q  NO3 CODIGO_ESTACAO
+    ## 1 2001-02-06 285.00034 0.27      MOGU02900
+    ## 2 2001-04-03 305.36021 0.28      MOGU02900
+    ## 3 2001-06-20 115.44747 0.47      MOGU02900
+    ## 4 2001-08-07  82.08674 0.58      MOGU02900
+    ## 5 2001-10-02 104.30206 0.58      MOGU02900
+
+Once the input parameters are set correctly, you can source the script. It is configured to run with the included example data to start.
+
+``` r
+source('batch.R')
+```
+
+Behind the scenes
+-----------------
+
+The script reads, processes, and writes output for each site/consituent combination individually. Currently, it runs three models for each iteration — the `rloadest` 5-parameter regression model, an interpolation model from `loadflex`, and a composite interpolation-regression model also from `loadflex`. Additional models can be added with some modifications to the script.
+
+All the site metadata is stored in `loadflex::metadata` objects, where it is referenced throughout the script.
+
+If the drainage basin areas for paired water quality and discharge sites are different (in the site metadata file, e.g `siteInfo.csv`), discharge is scaled by the appropriate ratio.
```

Comment:
add: "The resulting estimates describe fluxes at the water quality monitoring site."